dotcom-rendering/src/palette.ts

```diff
@@ -4990,6 +5066,18 @@ const paletteColours = {
 		light: recommendationCountArrowSelectedLight,
 		dark: recommendationCountArrowSelectedDark,
 	},
+	'--discussion-colour': {
```

Comment:
I think we generally avoid the word “colour“, as it does not bring much information, maybe we should favour “text” or “background” or simply drop it.

Reply:
`--discussion` sounds fine to me, but I see your point!